api_server/docker/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:22.04
+
+#######################################
+# Dockerfile for production !
+#######################################
+
+# Var for labels
+ARG GITHUB_ACTOR
+ARG GITHUB_REPOSITORY
+ARG GITHUB_REF
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Europe/Paris
+
+ENV PROJECT_NAME="PurpleCaffeine"
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV env="prod"
+
+LABEL org.opencontainers.image.title="${PROJECT_NAME}" \
+      org.opencontainers.image.authors=${GITHUB_ACTOR} \
+      org.opencontainers.image.vendor=${GITHUB_REPOSITORY} \
+      org.opencontainers.image.source="https://github.com/${GITHUB_ACTOR}/purplecaffeine" \
+      org.opencontainers.image.url="https://github.com/${GITHUB_ACTOR}/purplecaffeine/tags" \
+      org.opencontainers.image.description="API for quantum experiment tracking." \
+      org.opencontainers.image.os="Ubuntu" \
+      org.opencontainers.image.version=${GITHUB_REF}
+
+RUN apt-get update -yq \
+&& apt-get install python3-pip -y
+
+COPY api_server/code/ /opt/api_server/core
+COPY api_server/purplecaffeine /opt/api_server/purplecaffeine
+COPY api_server/requirements.txt /opt/api_server/
+COPY api_server/docker/start.sh /opt/
+
+RUN pip3 install --upgrade pip setuptools
+RUN pip3 install -r /opt/api_server/requirements.txt
+
+WORKDIR /opt/api_server
+
+EXPOSE 8000
+
+# I don't know how to launch Django without the webserver
```

Comment:
I'm usually going for `gunicorn` https://docs.djangoproject.com/en/4.2/howto/deployment/wsgi/gunicorn/

Basically run `gunicorn --bind 0.0.0.0:8000 myproject.wsgi`

Reply:
I just add this Dockerfile for preparing, but the Dockerize for production is in another issue #16